qa/rolling-upgrade/src/test/resources/processor/IndexMappings.json

```diff
@@ -16,7 +16,7 @@
         "method": {
           "name": "hnsw",
           "space_type": "l2",
-          "engine": "lucene",
+          "engine": "faiss",
```

Comment:
Last when I wrote BWC tests with "faiss" engine tests were not passing. Because test framework works well only with "lucene" engine. Can you run the test standalone and verify it once?

Reply:
This was changed before we had support for method parameters for lucene (it would have thrown errors with lucene). can roll it back now that we have support for lucene